tests/input/sla/the_double_sla.sla

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SCRIBUSUTF8NEW Version="1.4.6">
```

Comment:
Is this a file with.images inside ? If not, please change the file for a more complete one